Source/towners.cpp

```diff
@@ -742,6 +742,9 @@ void TalkToGirl(PlayerStruct &player, TownerStruct &girl)
 	case QUEST_ACTIVE:
 		InitQTextMsg(TEXT_GIRL3);
 		return;
+	case QUEST_DONE:

```

Comment:
Won't this result in the girl not saying thank you after getting the bear?

Reply:
this saying should be triggered by the first if - condition, isn't? surprisingly the second part is not in an `else` so i would have expected that right after finishing the quest the girl shoud have started crying.

Reply:
Yeah that is what it looks like.

Reply:
Just tested it, she cries but the text is correct...

Reply:
which version have you tested? my fix? I also tested it and she was not crying 🤔

Reply:
current master.

Reply:
then everything is fine